app/helpers/application_helper.rb

```diff
@@ -1214,7 +1215,7 @@ def start_page_allowed?(start_page)
   end
 
   def allowed_filter_db?(db)
-    return false if db.start_with?('Container') && !get_vmdb_config[:product][:containers]
+    return false if (db.nil? || db.start_with?('Container')) && !get_vmdb_config[:product][:containers]
     true
```

Comment:
@abonas : Good to know, it's going to get fixed.

However to review your change anyone has to figure out what this method does to understand what your change does.

So generally when dealing with such messy bit either don't touch it at all or clean it up, please.